### PR TITLE
Add some services

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/issues/346 https://github.com/AdguardTeam/HostlistsRegistry/issues/345 https://github.com/AdguardTeam/HostlistsRegistry/issues/344 https://github.com/AdguardTeam/HostlistsRegistry/issues/343 https://github.com/AdguardTeam/HostlistsRegistry/issues/341 https://github.com/AdguardTeam/HostlistsRegistry/issues/340 https://github.com/AdguardTeam/HostlistsRegistry/issues/339 https://github.com/AdguardTeam/HostlistsRegistry/issues/333 https://github.com/AdguardTeam/HostlistsRegistry/issues/332 https://github.com/AdguardTeam/HostlistsRegistry/issues/319

### CODE REVIEW

File: services/bigo_live.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...0 .3.2.3.9 0 1.9.1 2.8-.1 1-.3 1.6-1.1 1.6-2.1-.1-.9-.8-1.6-1.9-1.7h-2.4c-.3 0-.3.1-.3.3v1.7zm31.3 6.2v-2.6c.1-.3.3-.3.5-.3h6.4c.4 0 .6.2.6.6v7c0 .4-.2.6-.5.8a14.4 14.4 0 0 1-14.1-.1 9.3 9.3 0 0 1 .8-15.9c3.5-1.8 7.1-1.8 10.7-.8l2.1.7c.4.1.6.4.6.8V76c0 .7-.2.8-.9.5a15 15 0 0 0-6.9-1.5 6 6 0 0 0-4.9 2.5c-2.5 3.4-.7 8 3.4 8.9 1.6.4 3.2.3 4.8-.2.3-.1.4-.2.4-.6v-1.5c0-.3-.1-.4-.4-.4h-2.1c-.2 0-.5.1-.6-.3.2.1.2 0 .1-.1zM195 81v-8.6c0-.8 0-.8.8-.8h11.3c.8 0 .8 0 .8.8v2c0 .6-.2.7-.7.7H200c-.4 0-.6.1-.6.5v2.9c0 .4.1.5.5.5h7c.5 0 .7.2.7.7v2c0 .8-.1.8-.8.8H200c-.5 0-.6.1-.6.6v3.2c0 .4.1.5.5.5h7.3c.9 0 .9 0 .9.9v2c0 .4-.2.6-.6.6h-12c-.6 0-.6-.3-.6-.7l.1-8.6zm-10.8 1.5 2.5-5.7 2.1-4.8c.2-.4.4-.5.8-.5h3.2c.8 0 .9.2.5.9L187 86.2l-1.7 3.8c-.2.4-.4.5-.8.5-.9.1-1.3-.2-1.7-1l-8-17c-.4-.8-.3-1 .6-1h2.9c.5 0 .8.2 1 .6l4.6 9.9.3.5zm-26.4-10v13.3c-.3.2-.3.5 0 .7.1.4.3.2.5.2h7.9c.3 0 .6-.1.7.3v.3c-.1.1-.1.2 0 .2v2.2c0 .3-.2.4-.4.4h-12.8l-.1-.1V72c.1-.4.2-.6.6-.6h2.2c.3 0 .7.1 1-.1.5 0 .5.4.4.7-.2.4-.2.5 0 .5zm15.4 8.5v8.6c0 .7-.1.8-.8.8h-2.9c-.6 0-.7-.2-.7-.8V73c0-1.6-.1-1.4 1.4-1.5h2.3c.5 0 .7.2.7.7V81zm-75.3 0v8.5c0 .7 0 .8-.8.8h-2.9c-.7 0-.8 0-.8-.8V72.2c0-.5.2-.7.7-.7h3c.6 0 .7.2.7.7l.1 8.8z"/></svg>
```
Comment (Alex-302) [inline services/bigo_live.yml:8]: Better try to find something like that
https://www.svgrepo.com/svg/515017/bigo-live

File: services/plenty_of_fish.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
...-.63-.09-1.1-.26-1.42a.96.96 0 0 0-.85-.52c-.3-.02-.67.09-1.11.31-.48.24-.92.54-1.32.9v9.21c0 .42.1.84.3 1.22.2.42.48.8.81 1.13v.09h-6.16v-.1Zm-94.93-8.81c0 1.4.16 2.56.47 3.51.32.95.76 1.67 1.32 2.16.57.49 1.3.75 2.06.73.68 0 1.35-.17 1.93-.51.61-.34 1.1-.86 1.41-1.48l.18.09a7.44 7.44 0 0 1-1.19 2.65 5.05 5.05 0 0 1-4.18 2.19 4.7 4.7 0 0 1-2.08-.49 5.89 5.89 0 0 1-1.86-1.4 6.66 6.66 0 0 1-1.33-2.32 11.46 11.46 0 0 1-.03-6.5 8.9 8.9 0 0 1 1.3-2.7 6.5 6.5 0 0 1 1.99-1.84c.75-.44 1.6-.67 2.48-.67.9 0 1.79.27 2.52.8a5.58 5.58 0 0 1 1.78 2.14c.44.9.66 1.96.66 3.17l-9.48 2.76v-1.07l5.49-1.54a27.92 27.92 0 0 0-.2-3.16 4.82 4.82 0 0 0-.53-1.67c-.24-.38-.53-.57-.88-.57-.32 0-.62.2-.9.63-.28.42-.5 1.05-.67 1.89-.17.84-.26 1.91-.26 3.2ZM13.96 4.88C11.3.68 7.04 1.25.22 1.25v.09c.52.2.95.59 1.21 1.08.26.47.38 1.04.38 1.72v16.44c.01.61-.11 1.21-.38 1.76-.25.5-.68.9-1.2 1.1v.09h7.13v-.09c-.5-.22-.92-.6-1.17-1.1a3.78 3.78 0 0 1-.4-1.76V16c.54.13 1.08.2 1.62.19a7.66 7.66 0 0 0 6.84-4.3c.47-.97.7-2.1.7-3.41a6.6 6.6 0 0 0-1-3.6ZM9.45 13.6a3.33 3.33 0 0 1-2.96 1.65c-.24 0-.49-.02-.73-.05V7.56l.01-4.63a3 3 0 0 0-.1-.88c1.62 0 2.43.5 3.16 1.19.74.68 1.86 2.68 1.86 5.52a9.4 9.4 0 0 1-1.24 4.85Z"/></svg>
```
Comment (Alex-302) [inline services/plenty_of_fish.yml:5]: It has another icon
image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by jellizaveta** [comment]
New code was recently added. It can be pulled up already and it can be checked how the `services.json` file is built and check the validity of the svg.

**Comment by Alex-302** [comment]
@jellizaveta Checked, all good. Just 2 icons can be replaced.

File: services/plenty_of_fish.yml (hunk 2 above)
Comment (Sergey-Lyapin) [reply services/plenty_of_fish.yml]: It is ugly, I made another one.

**Review by Alex-302 — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

**Pull request merged**